src/components/Dropdown/asyncDropdown/AsyncDropdown.jsx

```diff
@@ -44,23 +47,26 @@ class AsyncDropDown extends React.Component {
 
   async handleInputChange(evt) {
     const { value } = evt.target;
-    const { currentDebounce } = this.state;
+    const { currentDebounce, page } = this.state;
 
     if (currentDebounce) {
       currentDebounce.cancel();
     }
     const debounced = debounce(() => this.handleSearch(value), 400);
     debounced();
     this.setState({
+      page: page ? 1 : undefined,
```

Comment:
blir det rett at page kan være undefined? (har ikke satt meg inn i komponenten bare stusset)

Reply:
Ja, bruker page som indikator for om man skal vise paginator eller ikke, men kanskje det er mer oversiktelig hvis den endres til en boolean og så trenger ikke state.page være undefined🤔